src/raftstore/store/worker/split_check.rs

```diff
@@ -237,16 +237,16 @@ impl<C: Sender<Msg>> Runner<C> {
                 .ch
                 .try_send(new_split_region(region_id, region_epoch, split_keys));
             if let Err(e) = res {
-                warn!("[region {}] failed to send check result: {}", region_id, e);
+                warn!("failed to send check result"; "region" => region_id, "error" => %e);
```

Comment:
we may need to unify field names later, i.e. `region_id` vs `region`. @huachaohuang